eng/FlakyTests.BeforeArcade.props

```diff
@@ -1,18 +0,0 @@
-<Project>
-  <!-- Local Dev Flakiness -->
-  <PropertyGroup>
-    <_FlakyRunAdditionalArgs>-trait "Flaky:All=true"</_FlakyRunAdditionalArgs>
-    <_NonFlakyRunAdditionalArgs>-notrait "Flaky:All=true"</_NonFlakyRunAdditionalArgs>
-  </PropertyGroup>
-
-  <!-- Azure Pipelines Flakiness -->
-  <PropertyGroup Condition="'$(TF_BUILD)' != ''">
```

Comment:
Just curious: Why are we removing platform-specific exclusions?

Reply:
Because a quarantined test is a quarantined test, doesn't matter what platform/configuration.

Putting specific exclusions is more of a way of putting off the issue instead of actually fixing it.

Reply:
:ok: